tests/trickle/main.c

```diff
@@ -31,36 +31,37 @@
 #define FIRST_ROUND     (5)
 #define SECOND_ROUND    (12)
 
-static uint32_t prev_now = 0, prev_diff = 0;
+static uint32_t old_t = 0;
```

Comment:
I find the name `old_t` a little bit confusing. I read the code before I read the definitions and was confused that *a type* got assigned a value.

Sorry for the late look at this, but I fell a little behind on my mails ;-).